coro/coro.h

```diff
@@ -168,6 +168,8 @@ extern "C" {
 
 #include <stddef.h>
 
+#include <evfibers/config.h>
```

Comment:
The coro.h file is a 3rdparty header (libcoro), it's incorrect to include evfibers/* in it. If you want to define something for libcoro, please do so via `add_definition()` in CMake.